copypasta/src/windows.rs

```diff
@@ -0,0 +1,75 @@
+use clipboard::ClipboardContext;
+use clipboard::ClipboardProvider;
+
+use super::{Load, Store};
+
+pub struct Clipboard(ClipboardContext);
+
+#[derive(Debug)]
+pub enum Error {
+    Clipboard(Box<::std::error::Error>),
+}
+
+impl ::std::error::Error for Error {
+    fn description(&self) -> &str {
+        match *self {
+            Error::Clipboard(..) => "error opening clipboard",
+        }
+    }
+}
+
+impl ::std::fmt::Display for Error {
+    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        match *self {
+            Error::Clipboard(ref err) => err.fmt(f),
+        }
+    }
+}
+
+// TODO: Is this really safe?
```

Comment:
This should probably be properly evaluated before merging. I haven't taken a close at it myself yet, just leaving a comment for tracking here.